src/miner.cpp

```diff
@@ -731,7 +731,7 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn,  unsigned int nBlo
 
         // Fill in header
         pblock->hashPrevBlock  = pindexPrev->GetBlockHash();
-        pblock->hashScTxsCommitment = pblock->BuildScTxsCommitment();
+        pblock->hashScTxsCommitment = CPoseidonHash(pblock->BuildScTxsCommitment());
```

Comment:
`pblock->BuildScTxsCommitment()` may return `CPoseidonHash` type.

Reply:
CPoseidonHash is a temporary wrapper until 256-bit Poseidon Hash will be introduced.